app/views/practices/show/implementation/_resource_link_attachment.html.erb

```diff
@@ -2,10 +2,12 @@
 <ul class="usa-list--unstyled margin-bottom-5">
   <% link_attachments.each do |la| %>
     <div class="margin-bottom-1">
-      <% if la.name.present? %>
+      <% if la.link_url.present? %>
         <% unless la.link_url.start_with?("http") %>
           <% la.link_url = "http://" + la.link_url %>
         <% end %>
+        <% la.name = la.link_url unless la.name.present? %>
```

Comment:
I don't think we need `line 9-10` we can just do `<%= la.name || la.link_url %>` for `<%= la.name %>` on line 12

Reply:
Got it Thx Elisa.